.tekton/release-service-pull-request.yaml

```diff
@@ -174,35 +161,34 @@ spec:
         values:
         - "true"
       workspaces:
-      - name: output
-        workspace: workspace
       - name: basic-auth
         workspace: git-auth
     - name: prefetch-dependencies
       params:
       - name: input
         value: $(params.prefetch-input)
+      - name: SOURCE_ARTIFACT
+        value: $(tasks.clone-repository.results.SOURCE_ARTIFACT)
+      - name: ociStorage
+        value: $(params.output-image).prefetch
+      - name: ociArtifactExpiresAfter
+        value: $(params.image-expires-after)
       runAfter:
       - clone-repository
       taskRef:
         params:
         - name: name
-          value: prefetch-dependencies
+          value: prefetch-dependencies-oci-ta
```

Comment:
prefetch-dependencies. Lets keep the name to original to pass the EC checks

Reply:
Original name?  It's the  task name in the bundle of "quay.io/konflux-ci/tekton-catalog/task-prefetch-dependencies-oci-ta".  And it's being used in the tekton files for integration-service and build-definitions [here](https://github.com/flacatus/build-service/blob/4a98c2e92733bd1d1bc20fe47402ede1c1f34e32/.tekton/build-service-pull-request.yaml#L214C18-L214C46).

Reply:
Found and fix the issue - the version of the bundle need to change to "0.2" .